charts/partials/templates/_proxy-init.tpl

```diff
@@ -47,7 +47,11 @@ securityContext:
     {{- end }}
     {{- end }}
   {{- if or .Values.proxyInit.closeWaitTimeoutSecs .Values.proxyInit.runAsRoot }}
+  {{- if .Values.proxyInit.closeWaitTimeoutSecs }}
```

Comment:
I'm not sure, but should this be `{{- if or .Values.proxyInit.closeWaitTimeoutSecs .Values.proxyInit.runAsRoot }}` like on line 32?

Reply:
@adleong No, that is exactly the problem addressed by this PR, please see #7400 for more details.